<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>756</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(220, 114, 160, 255), stop:1 rgba(240, 207, 211, 255));
}
</string>
  </property>
  <widget class="QPushButton" name="l1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>450</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#l1{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(240,167,98,219), stop:1 rgba(85, 98, 112, 226));
color:rgba(255,255,255,210);
border-radius:10px;
}
QPushButton#l1:hover{
background-color: qlineargradient(spread:pad,x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(212, 226, 128, 200), stop:1 rgba(105, 118, 132, 226));
color:rgba(255,255,255,210);
border-radius:10px;
}


QPushButton#l1:pressed{
padding-left:4px;
padding-top:4px;

}
</string>
   </property>
   <property name="text">
    <string>L O G I N</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="u1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,0);
border-radius:8px;
border:none;
border-bottom:3px solid rgb(105,118,132,255);
color: rgb(235, 235, 235);
padding-bottom:5px;
</string>
   </property>
   <property name="placeholderText">
    <string>Enter the username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="p1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0,0,0,0);
border:none;
border-radius:8px;
border-bottom:3px solid rgb(105,118,132,255);
color: rgb(235, 235, 235);
padding-bottom:5px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter the password</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="signIn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>590</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#signIn{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(80, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
color:rgba(255,255,255,210);
border-radius:10px;
}
QPushButton#signIn:hover{
background-color: qlineargradient(spread:pad,x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40,67,98,219), stop:1 rgba(105, 118, 132, 226));
color:rgba(255,255,255,255);
border-radius:10px;
}

QPushButton#signIn:pressed{
padding-left:3px;
padding-top:3px;
}
</string>
   </property>
   <property name="text">
    <string>Sigin In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;kknkn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255,255);</string>
   </property>
   <property name="text">
    <string>New user ? Sign in</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
